org.ekstep.reviewercomments-1.0/editor/plugin.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Plugin to get comments from portal
+ * @class reviewerComments
+ * @constructor
+ * @extends org.ekstep.contenteditor.basePlugin
+ * @author Revathi P 
+ * @listens stage:select
+ */
+
+org.ekstep.contenteditor.basePlugin.extend({
+    initialize: function() {
+        ecEditor.addEventListener("stage:select", this.initShowComments, this);
+    },
+    initShowComments: function(event, data) {
```

Comment:
`initShowComments ` is not proper

Reply:
@manjudr : The mock api is updated in the wiki document, https://project-sunbird.atlassian.net/wiki/spaces/SBDES/pages/678035590/Design+Display+the+page+wise+comments+for+content+editor. Please check.